### ISSUE TITLE
Import/update PyMaemo

### ISSUE THREAD

**Issue by MerlijnWajer** [maemo-leste/bugtracker#260]
PyMaemo has a lot of `python-` packages of the Hildon APIs. What better way to expose the power of Maemo (Leste) than by having the Python packages available too?

We have a git-svn clone of this repository https://vcs.maemo.org/svn/pymaemo/packages/ here: https://github.com/maemo-leste/pymaemo-svn

Anyone feel like picking this up?

**Comment by MerlijnWajer** [comment]
Probably start with building Devuan pygtk with the hildon patches.
Then python-osso, python-hildon, python-hildondesktop ? (And any dependencies)

**Comment by MerlijnWajer** [comment]
`python-central` seems to be some hack that maemo developers used to pull in some other python version that was not the system version? I don't think we need this at all right now. I'm going to work assuming that the default python version is 2 (2.7), and see where things break.

Eventually we'll have to go to 3.x, but let's get 2.7 working first, right? :)

**Comment by sicelo** [comment]
>Eventually we'll have to go to 3.x, but let's get 2.7 working first, right? :)

yes, let's keep 2.7 as long as possible, since that's what the python applications in the repositories use.

**Comment by MerlijnWajer** [comment]
It's mostly working now.

https://wizzup.org/python-maemo-binaryclock.png

**Comment by MerlijnWajer** [comment]
Also see #266
